default.json

```diff
@@ -23,6 +26,14 @@
       "groupName": "Voiceflow dependencies",
       "matchPackagePrefixes": ["@voiceflow/"],
       "enabled": true
+    },
+    {
+      "groupName": "Voiceflow dependencies (automerge)",
+      "matchPackagePrefixes": ["@voiceflow/"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "matchCurrentVersion": "!/^0/",
```

Comment:
renovate understands semver, this shouldn't be necessary

Reply:
I grabbed this from here: https://docs.renovatebot.com/key-concepts/automerge/#automerge-non-major-updates

> The `matchCurrentVersion` setting above is a rule to exclude any dependencies which are pre-1.0.0 because those can make breaking changes at any time according to the SemVer spec.

Reply:
hm okay, i must be mistaken then